bluepill/tests/BPIntegrationTests.m

```diff
@@ -135,7 +135,17 @@ - (void)testTwoBPInstancesWithXCTestRunFile {
     BPRunner *runner = [BPRunner BPRunnerWithConfig:self.config withBpPath:bpPath];
     XCTAssert(runner != nil);
     int rc = [runner runWithBPXCTestFiles:app.testBundles];
-    XCTAssert(app.testBundles[1].skipTestIdentifiers.count == 8);
+    if (app.testBundles[1].skipTestIdentifiers.count != 8) {
+        NSLog(@"testBundles");
+        for (int i = 0; i < app.testBundles.count; ++i) {
+            NSLog(@"%d - %@", i, app.testBundles[i].name);
+            for (NSString *s in app.testBundles[i].skipTestIdentifiers) {
+                NSLog(@"\t%@", s);
+            }
+        }
+        XCTAssert(FALSE, @"Sorry kiddo, you lose");
+
```

Comment:
It's not clear from this assertion as to what failed, unless we take a look into the code. Whereas earlier, it used to clearly say skipTestIdentifiers' count is not right.
How about we keep the same XCTAssert right after this debug log, even though it is less cool?  
Plus there's an extra blank line here.

Reply:
This is not intended to be committed, we are trying to figure out why this test suddenly started failing in CI when it had been passing.

Reply:
Cool, thanks.